.cirrus.yml

```diff
@@ -0,0 +1,27 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+task:
+  name: FreeBSD
+  env:
+    matrix:
+      - JULIA_VERSION: 1.3
+        CUDA: nill
+      - JULIA_VERSION: nightly
+        CUDA: nill
+      - JULIA_VERSION: 1.3
+        CUDA: 8.0
+      - JULIA_VERSION: nightly
+        CUDA: 8.0
+      - JULIA_VERSION: 1.3
+        CUDA: 10.1
+      - JULIA_VERSION: nightly
+        CUDA: 10.1
+  install_script:
+    - ./.cirrus.linux
```

Comment:
Does this script work on FreeBSD?

Reply:
Cirrus isn't enabled, and I don't have access to a FreeBSD system otherwise, so I don't know yet